scraper.rb

```diff
@@ -24,6 +24,10 @@ def cell(profile, th)
   td.text.strip
 end
 
+def multiple_values(values)
```

Comment:
The name of this function isn't particularly clear. (Or, from the other side, faced with a function called this, this isn't the implementation I'd expect to see in it)